packages/pigment-css-react/src/processors/sx.ts

```diff
@@ -99,6 +99,19 @@ export class SxProcessor extends BaseProcessor {
         ([variableId, expression, isUnitLess]) => {
           switch (expression.type) {
             case 'ArrowFunctionExpression': {
+              const fnBody = expression.body as Expression;
+              // try to deserialize AST if possible
+              if (
```

Comment:
Do we even need this condition here? We anyway have the code wrapped in try & catch.

Reply:
Removed other checks but `StringLiteral` check is needed for type narrowing.